src/apps/setup.ts

```diff
@@ -21,7 +21,7 @@ export const setupAppFactory = (
     // If not on Glitch or Production, create a smee URL
     if (
       process.env.NODE_ENV !== "production" &&
-      !(process.env.PROJECT_DOMAIN || process.env.WEBHOOK_PROXY_URL || host)
+      !(process.env.PROJECT_DOMAIN || process.env.WEBHOOK_PROXY_URL || process.env.NO_SMEE_SETUP === 'true')
```

Comment:
I don't like this hack, but it's probably better than reverting to the previous behavior. At least you are unblocked. But we need to find a better way to make probot work with your environment than an undocumented environment variable. I can't make promises that this behavior won't change between bug or feature releases, so maybe better pin your probot version

Reply:
You are right, would you prefer that I add documentation to make it documented or did you have something completely different in mind?
Please let me know if this would make it better: https://github.com/kammerjaeger/probot/tree/docFor_NO_SMEE_SETUP

Reply:
> You are right, would you prefer that I add documentation to make it documented or did you have something completely different in mind?

probably something entirely different

> The correct way to do it would probably be that if GHES is used, Smee should be disabled by default, most cooperation don't like it if data is leaving the network

that is a good point, we might do that. 

For now I'll create a follow up issue